docs/clients/p4rt-ctl.rst

```diff
@@ -91,6 +91,24 @@ services that are available for a p4runtime client.
 For information on security and enabling TLS on gRPC ports, refer to
 guides/security-guide.md
 
+Usage: p4rt-ctl [OPTIONS] COMMAND [ARG...]
```

Comment:
The usage text renders badly. I think you need to make it a code block.

(You can visualize it by selecting the **View file** option from the ellipsis menu in github)

Reply:
Fixed it

Reply:
Still looks bad.

I think you need to specify a blank line between `code-block` and the display text, and you need to indent each of the lines in the display text:

```text
.. code-block:: text

  Usage: p4rt-ctl [OPTIONS] COMMAND [ARG...]

  positional arguments:
    command               Subcommand to run

  optional arguments:
    -h, --help            show this help message and exit
    -g, --grpc_addr       GRPC_ADDR P4Runtime gRPC server address, format : <server IP>:<port>
```
It looks okay (in VS Code preview) when I do this.

Reply:
The indentation kind of makes sense. Without the change from indented to unindented, how does the renderer know where the code block ends?

Reply:
Adding the indentation did the trick. Thanks Derek!